src/gui/test/components/navigation/footer.tsx

```diff
@@ -0,0 +1,81 @@
+import { afterEach, it, describe, vi, expect } from 'vitest'
+import { cleanup, render, screen } from '@testing-library/react'
+import html from 'diffable-html'
+import { useGraphStore as useStore } from '@/state/index.js'
+import { Footer } from '@/components/navigation/footer.jsx'
+
+vi.mock('useTheme', () => ({
+  useTheme: vi.fn(),
+}))
+vi.mock('useGraphStore', () => ({
+  useGraphStore: vi.fn(),
+}))
```

Comment:
same as before, I don't think we need / should be mocking these, is `useTheme` even still a thing?